chapter2/tests/t1/chapter2-t1-two-files.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# stop on first error
+set -e
+
+# validate
```

Comment:
Please add docs in a format of a styleguide - https://google.github.io/styleguide/shellguide.html